### DISCUSSION TITLE
Add WGSL Support

### DISCUSSION THREAD

**Discussion by pati08** [boyter/scc#456] [Ideas]
WGSL (WebGPU Shading Language) would be a nice thing to have added. It seems pretty simple to add; from what I can tell it doesn't even use strings. I think the only consideration would be comments, which seem to me to just be c-like comments. I'd add it myself and PR, but I don't know Go.

**Comment by boyter** [comment]
Good news! You don't need to know Go to do it. See https://github.com/boyter/scc/blob/master/languages.json as the definition for each type.

Otherwise post a link to the language definition here and ill hook it up.

**Comment by pati08** [comment]
Cool, thanks! I'll probably submit a PR today

**Comment by pati08** [comment]
Addressed by #458

**Comment by boyter** [comment]
Merged! You will need to wait for a new release or  build to get it, but thankfully there is a bug I am working on so you might get it in a few days.

**Comment by boyter** [comment]
Latest release 3.3.4 should have this for you.

**Comment by pati08** [comment]
Awesome, thanks!
